woocommerce/woocommerce-gutenberg-products-block
# Cart i2: Toolbar based view switcher

Implements an updated View Switcher as a toolbar item when viewing the cart block, and filled cart/empty cart inner blocks. These components are shared using context.

This has not had design input, but using the toolbar was a suggestion in p2. I've used a dropdown toolbar item to make it align with other controls. I think this should be ok to help us make progress until it's had a full design review because it's behind the experimental flag.

Part of #4793
Fixes #4794

### Screenshots

![Screenshot 2021-09-22 at 14 14 16](https://user-images.githubusercontent.com/90977/134350835-189ee19b-8913-4387-aa29-307182ca8a93.png)

### How to test the changes in this Pull Request:

1. Insert cart i2
2. Select either the top-level cart i2 block, or the filled/empty inner block
3. Try to switch view
4. View should change. Switcher should reflect the updated state.
